docs/backup-restore.md

```diff
@@ -5,19 +5,41 @@ This document sets out some guidelines for backing up and restoring the database
 
```

Comment:
Can you exclude the (non-related) changes from other PRs?
I have several comments on the backup guide.

Reply:
Sorry, I got it.
I will arrange the branch after https://github.com/scalar-labs/scalardb/pull/761 is merged.

Reply:
I fixed the commit logs. I'll merge this PR after the CI passes.